install/redmine/install.yml

```diff
@@ -0,0 +1,15 @@
+id: redmine
+name: Redmine
+title: Redmine monitoring
+description: |
+  New Relic’s instant observability quickstart with Redmine monitoring helps, you get full visibility into the complete webpage life cycle of your application or website.
```

Comment:
"New Relic’s instant observability Redmine quickstart helps you get full visibility into the into the performance of your Redmine app."